java/test/jmri/jmrit/logix/WarrantFrameTest.java

```diff
@@ -93,20 +92,21 @@ public void testJoinWarrantsNoStop() throws Exception {
         Warrant startW = _warrantMgr.getWarrant("WestBoundStart");
         Warrant endW = _warrantMgr.getWarrant("WestBoundFinish");
 
-        new Thread(() -> {
-            JFrameOperator jfo = new JFrameOperator(WarrantTableFrame.getDefault());
-            JDialogOperator jdo = new JDialogOperator(jfo, Bundle.getMessage("QuestionTitle"));
-            JButtonOperator jbo = new JButtonOperator(jdo, Bundle.getMessage("ButtonNo"));
-            jbo.push();
-        }).start();
-        
+        Thread clickDialog = new Thread(() -> {
```

Comment:
No need to change, but the code might be easier to read if `clickDialog` was renamed to `clickDialogThread`.